kvs/src/monitor/stats_helpers.cpp

```diff
@@ -97,6 +105,28 @@ void collect_internal_stats(
               key_access_frequency[key][ip_pair + ":" + std::to_string(tid)] =
                   key_count.access_count();
             }
+          } else if (metadata_type == "hot_access") {
+            // deserialized the value
+            KeyAccessData access;
+            access.ParseFromString(lww_value.value());
+
+            for (const auto& key_count : access.keys()) {
+              Key key = key_count.key();
+              hot_key_access_frequency[key]
```

Comment:
Could this be extracted into a method with the cold_key_access_frequency or hoy_key_access_frequency be passed in?

Reply:
+1